include-build/roborazzi-gradle-plugin/src/integrationTest/java/io/github/takahirom/roborazzi/RoborazziGradleProject.kt

```diff
@@ -228,24 +228,24 @@ android {
 dependencies {
   implementation("io.github.takahirom.roborazzi:roborazzi:1.0.0")
```

Comment:
Why not include the local libraries in the catalog?

Reply:
I thought it would be confusing to include the roborazzi 1.0.0 declaration.
However, I will include it.